code/modules/paperwork/fax.dm

```diff
@@ -255,6 +262,28 @@
 				loaded_item_ref = null
 				update_appearance()
 				return TRUE
+		
+		if("send_special")
+			if(!istype(loaded_item_ref?.resolve(), /obj/item/paper))
+				to_chat(usr, icon2html(src.icon, usr) + span_warning("Fax cannot send all above paper on this protected network, sorry."))
+				return 
+			
+			var/obj/item/paper/fax_paper = loaded_item_ref?.resolve()
+			fax_paper.request_state = TRUE
```

Comment:
What does this do?

Reply:
get the paper from inputed loaded_item_ref, and give ability reard it from everywhere